test/widget/auth_widget_test.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:crowdleague/models/app/app_state.dart';
+import 'package:crowdleague/reducers/app_reducer.dart';
+import 'package:crowdleague/widgets/auth/auth_page.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_redux/flutter_redux.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:redux/redux.dart';
+
+import '../mocks/image_test_utils.dart';
+
+void main() {
+  group('auth page', () {
+    testWidgets('displays without overflowing', (WidgetTester tester) async {
+      // passing test indicates no overflowing as test suite uses a small device screen
+
+      // Setup the app state with expected values
+      final initialAppState = AppState.init();
+      // Create the test harness.
+      final store = Store<AppState>(appReducer, initialState: initialAppState);
+      final wut = AuthPage();
+      final harness =
+          StoreProvider<AppState>(store: store, child: MaterialApp(home: wut));
+
+      await provideMockedNetworkImages(() async {
```

Comment:
this is my bad but could we try without the provideMockedNetworkImages wrapper?  Thinking about it now I reckon it's worth only adding when needed

Reply:
shweeeet